doc/architecture/decisions/0001-use-problem-responses-to-handle-validation-errors.md

```diff
@@ -0,0 +1,59 @@
+# 1. Use `application/problem+json` responses to handle validation errors
+
+Date: 2022-07-25
+
+## Status
+
+Accepted
+
+## Context
+
+We have already decided to
+[handle validation errors at the API level][1]
+, but we need a to agree a common and predictable way to return validation or other
+errors to the client, so they can be parsed an returned to the user.
+
+## Decision
+
+We will return `application/problem+json` responses, as outlined in [RFC-7807][2]. As well as returning the standard Problem
+response, we will also return details of what fields caused the error in an
+`invalid-params` array. For example:
```

Comment:
Is it worth referencing the "extension members" terminology of the RFC (https://datatracker.ietf.org/doc/html/rfc7807#section-3.2) ?